samples/packages/enrichment-migrator/src/main/kotlin/EnrichmentMigrator.kt

```diff
@@ -80,38 +90,102 @@ object EnrichmentMigrator {
             }
             start.toList()
         }
-        targetConnectionQNs.forEachIndexed { index, targetConnectionQN ->
-            val ctx = MigratorContext(
-                sourceConnectionQN = sourceConnectionQN,
-                targetConnectionQN = targetConnectionQN,
-                includeArchived = includeArchived,
-            )
-            val targetConnectionFilename = targetConnectionQN.replace("/", "_")
-            val transformedFile = "$outputDirectory${File.separator}CSA_EM_transformed_$targetConnectionFilename.csv"
-            val transformer = Transformer(
-                ctx,
-                extractFile,
-                header.toList(),
-                logger,
-                fieldSeparator,
-            )
-            transformer.transform(transformedFile)
+        targetConnectionQNs.forEachIndexed { _, targetConnectionQN ->
```

Comment:
Since we're not using the index anyway, should just replace `forEachIndexed` with `forEach` and drop the first arg altogether.